tests/integration.rs

```diff
@@ -120,6 +120,37 @@ fn invert() -> Result<()> {
   assert_output_eq(&["resize:1:1", "all"], "1")
 }
 
+#[test]
+fn save() -> Result<()> {
+  let path = "output.png";
```

Comment:
I would just put "output.png" in-line. I think it hurts readability a little bit.

```suggestion
```